lib/charms/operator_libs_linux/v0/apt.py

```diff
@@ -124,7 +131,9 @@
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 17
+LIBPATCH = 18
+
+PYDEPS = ["opentelemetry-api"]
```

Comment:
This charm uses `ops==3.0.0`, and thus already has this as a transient dependency.

The dependency could be added directly, especially if we want to pin the exact version.

https://discourse.charmhub.io/t/adding-to-tracing-to-charm-libraries-starting-with-apt/18260